enterprise/backup/src/test/java/org/neo4j/backup/OnlineBackupCommandCcIT.java

```diff
@@ -0,0 +1,228 @@
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.backup;
+
+import org.apache.commons.lang3.SystemUtils;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.IntFunction;
+import javax.annotation.Nullable;
+
+import org.neo4j.causalclustering.core.CausalClusteringSettings;
+import org.neo4j.causalclustering.core.CoreGraphDatabase;
+import org.neo4j.causalclustering.core.consensus.roles.Role;
+import org.neo4j.causalclustering.discovery.Cluster;
+import org.neo4j.causalclustering.discovery.CoreClusterMember;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Node;
+import org.neo4j.graphdb.RelationshipType;
+import org.neo4j.graphdb.Transaction;
+import org.neo4j.graphdb.factory.GraphDatabaseSettings;
+import org.neo4j.kernel.impl.enterprise.configuration.OnlineBackupSettings;
+import org.neo4j.kernel.configuration.Settings;
+import org.neo4j.kernel.impl.store.format.highlimit.HighLimit;
+import org.neo4j.kernel.impl.store.format.standard.Standard;
+import org.neo4j.test.DbRepresentation;
+import org.neo4j.test.causalclustering.ClusterRule;
+import org.neo4j.test.rule.SuppressOutput;
+import org.neo4j.util.JvmRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assume.assumeFalse;
+
+@RunWith( Parameterized.class )
+public class OnlineBackupCommandCcIT extends JvmRunner
+{
+    @Rule
+    public ClusterRule clusterRule = new ClusterRule( getClass() )
+            .withNumberOfCoreMembers( 3 )
+            .withNumberOfReadReplicas( 3 )
+            .withSharedCoreParam( CausalClusteringSettings.cluster_topology_refresh, "5s" );
+
+    @Rule
+    public final RuleChain ruleChain = RuleChain.outerRule( SuppressOutput.suppressAll() ).around( clusterRule );
+
+    private static final String ip = "127.0.0.1";
+
+    private File backupDir;
+
+    @Parameter
+    public String recordFormat;
+
+    @Parameters( name = "{0}" )
+    public static List<String> recordFormats()
+    {
+        return Arrays.asList( Standard.LATEST_NAME, HighLimit.NAME );
+    }
+
+    @Before
+    public void initialiseBackupDirectory()
+    {
+        backupDir = testDirectory.directory( "backups" );
+    }
+
+    @Test
+    public void backupCanBePerformedOverCcWithDefaultPort() throws Exception
+    {
+        assumeFalse( SystemUtils.IS_OS_WINDOWS );
```

Comment:
Why do all these tests skip Windows?

Reply:
That's what was in the original - not sure why